README.md

```diff
@@ -1,3 +1,244 @@
 # React Breakpoints Mixin
 
 BreakpointsMixin for your React Components.
+
+
+## Example
+
+See the working [example page](http://ui.deseretdigital.com/react-breakpoints-mixin/).
+
+
+## Install
+
+The package is available through npm and also bower.
+
+### npm
+
+```shell
+npm install --save react-breakpoints-mixin
+```
+
+
+### bower
+
+```shell
+bower install --save react-breakpoints-mixin
+```
+
+
+## Import
+
+The `BreakpointsMixin` module is packaged in the [UMD][GoogleUMD] format, which
+means you can bring it in as a [CommonJS][GoogleCommonJS] module using `require`,
+or as a [global variable][GoogleGlobalVariable] using a `<script>` tag.
+
+[GoogleUMD]: https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=javascript%20umd
+[GoogleCommonJS]: https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=commonjs+module
+[GoogleGlobalVariable]: https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=why%20javascript%20global%20variables%20are%20bad
+
+### CommonJS
+
+```javascript
+var BreakpointsMixin = require('react-breakpoints-mixin').BreakpointsMixin;
+```
+
+The export has a `BreakpointsMixin` property which you can assign to a variable.
+
+
+### Global Variable
+
+```javascript
+  <script src="bower_components/react-breakpoints-mixin/dist/react-breakpoints-mixin.js"></script>
+```
+
+The script exposes a `BreakpointsMixin` global variable.
+
+
+## Use
+
+Once you have imported the `BreakpointsMixin` variable, you need to ...
+
+1. Add it to your fancy component's list of mixins.
+2. Define breakpoints for width and/or height.
+3. Generate css classes using the provided `this.breakpointsClasses()` method.
+
+```javascript
+var FancyComponent = React.createClass({
+
+  mixins: [ BreakpointsMixin ], /* [1] */
+
+  breakpoints: { /* [2] */
+    width: {
+      "small": [0, 400],
+      "medium": [400, 600],
+      "large": [600, Infinity]
+    },
+    height: {
+      "short": [0, 100],
+      "medium": [100, 200],
+      "long": [200, Infinity]
+    }
+  },
+
+  render: function () {
+    <div class="this.getClasses()">Hello World</div>
```

Comment:
`<div className={this.getClasses()}>Hello World</div>`

Maybe I should have copied and pasted some code there?